met/src/basic/vx_log/string_array.cc

```diff
@@ -182,7 +187,7 @@ void StringArray::dump(ostream & out, int depth) const
 Indent prefix(depth);
 Indent prefix2(depth + 1);
 
-out << prefix << "IgnoreCase = " << IgnoreCase << "\n";
+out << prefix << "IgnoreCase = " << IgnoreCase << " Sorted = " << Sorted << "\n";
```

Comment:
Recommend following the convention in TimeArray::dump().
```suggestion
out << prefix << "IgnoreCase = " << IgnoreCase << "\n";
out << prefix << "Sorted     = " << (Sorted ? "true" : "false") << "\n";
```